src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -1102,14 +1102,19 @@ private void purge(List<String> files) {
     private void moveToTableStage(List<String> failedFiles) {
       if (!failedFiles.isEmpty()) {
         OffsetContinuityRanges offsets = searchForMissingOffsets(failedFiles);
-        LOGGER.info(
-            "Moving failed files for pipe: {} to tableStage failedFileCount: {}, continuousOffsets:"
-                + " {}, missingOffsets: {}",
-            pipeName,
-            failedFiles.size(),
-            offsets.getContinuousOffsets(),
-            offsets.getMissingOffsets());
-        LOGGER.debug("Moving failed files: {}", failedFiles);
+        String baseLog =
+            String.format(
+                "Moving failed files for pipe: %s to tableStage failedFileCount: %d,"
+                    + " continuousOffsets: %s, missingOffsets: %s",
+                pipeName,
+                failedFiles.size(),
+                offsets.getContinuousOffsets(),
+                offsets.getMissingOffsets());
+        if (LOGGER.isDebugEnabled()) {
+          LOGGER.info("{} - {}", baseLog, failedFiles);
```

Comment:
wouldn't `LOGGER.info("{}, failedFiles: {}")` be more consistent? I am hoping `List<String>` goes out as `[string0, string1, ...]`

Reply:
Changed to `LOGGER.info("{}, failedFiles: {}")`